ADOPTERS.md

```diff
@@ -25,7 +25,7 @@ publicly shared the details of how they use it.
 BitGo uses Velero backup and restore capabilities to seamlessly provision and scale fullnode statefulsets on the fly as well as having it serve an integral piece for our kubernetes disaster-recovery story.
 
 **[Bugsnag][30]**  
-We use Velero for managing backups of an internal instance of our on-premise clustered solution. We also recommend our users of [on-premise Bugsnag installations][31] use Velero for [managing their own backups][32].
+We use Velero for managing backups of an internal instance of our on-premises clustered solution. We also recommend our users of [on-premises Bugsnag installations][31] use Velero for [managing their own backups][32].
```

Comment:
While "on-premise" is technically wrong ("premises" refers to buildings, sites, etc. while "premise" is a proposition or idea), in this case, Bugsnag (as an adopter) uses "on-premise" in their own marketing materials on their site, so I don't think it's Velero's place to change that for them.